outputs.tf

```diff
@@ -1,7 +1,14 @@
-output "addon" {
-  value = {
-    name : "grafana-agent-operator"
-    version : "0.28.2"
-    content : local.yaml
-  }
+output "addons" {
```

Comment:
Nice! Did we finish "transforming" the other modules/addons to use `addons` instead of `addon` ?
If not perhaps we should create some issues to keep track?

Reply:
Might be a good idea. Haven't checked the others.

Reply:
Done!